src/hooks/content/useContent.js

```diff
@@ -4,18 +4,18 @@ import { useSelector, shallowEqual } from 'react-redux';
  * useContent hook
  *
  * This hook returns the current content that is stored in the Redux store in the
- * `content` reducer, and returns it along with the related state (loading/loaded/error).
+ * `content` reducer, and returns it along with the related state (loading/loaded/error/title).
  *
  * @export
- * @return {{ data: ContentData, loading: boolean, loaded: boolean, error: Error }}
+ * @return {{ data: ContentData, loading: boolean, loaded: boolean, error: Error,title }}
  */
 export function useContent() {
-  const data = useSelector((state) => state.content.data, shallowEqual);
-  const loading = useSelector((state) => state.content.get.loading);
-  const loaded = useSelector((state) => state.content.get.loaded);
-  const error = useSelector((state) => state.content.get.error, shallowEqual);
-
-  return { data, loading, loaded, error };
+  const data = useSelector((state) => state.content?.data, shallowEqual);
+  const loading = useSelector((state) => state.content.get?.loading);
+  const loaded = useSelector((state) => state.content.get?.loaded);
+  const error = useSelector((state) => state.content.get?.error, shallowEqual);
+  const title = useSelector((state) => state.content.data?.title, shallowEqual);
```

Comment:
I don't think we need an extra field for title, as we are exporting whole content.